filebeat/module/nginx/ingress_controller/test/test.log-expected.json

```diff
@@ -28,18 +28,30 @@
         "nginx.ingress_controller.upstream.alternative_name": "",
         "nginx.ingress_controller.upstream.ip": "172.17.0.5",
         "nginx.ingress_controller.upstream.name": "default-web-8080",
-        "nginx.ingress_controller.upstream.port": "8080",
-        "nginx.ingress_controller.upstream.response.length": 59,
+        "nginx.ingress_controller.upstream.port": 8080,
+        "nginx.ingress_controller.upstream.response.length_total": 59,
+        "nginx.ingress_controller.upstream.response.length_list": [
+            "59"
+        ],
         "nginx.ingress_controller.upstream.response.status_code": 200,
-        "nginx.ingress_controller.upstream.response.time": 0.0,
+        "nginx.ingress_controller.upstream.response.status_code_list": [
+            "200"
+        ],
+        "nginx.ingress_controller.upstream.response.time_total": 0.0,
+        "nginx.ingress_controller.upstream.response.time_list": [
+            "0.000"
+        ],
+        "nginx.ingress_controller.upstream_ip_list": [
```

Comment:
We are not storing ips in this field, maybe we can call it `upstream_endpoint_list`, or `upstream_address_list`.